src/ergw_aaa_diameter_srv.erl

```diff
@@ -179,7 +179,8 @@ call(App, Request, #{function := Function} = Config, CallOpts) ->
 	    case Error of
 		encode ->
 		    ?LOG(critical, "failed to encode DIAMETER requests: ~0p", [Request]);
-		_ ->
+		Other ->
+		    ?LOG(notice, "non-critical diameter API: ~0p", [Other]),
```

Comment:
Maybe
```suggestion
		    ?LOG(notice, "non-critical diameter API: ~p", [Other]),
```
?

Reply:
I assume the previous 0p is more critical since the request can be arbitrary long, so yes, maybe the 0p is not really needed here.